src/overlay/Peer.h

```diff
@@ -459,13 +445,8 @@ class Peer : public std::enable_shared_from_this<Peer>,
 
     void sendTxDemand(TxDemandVector&& demands);
     void fulfillDemand(FloodDemand const& dmd);
-    void queueTxHashToAdvertise(Hash const& hash);
-    void queueTxHashAndMaybeTrim(Hash const& hash);
-    TxAdvertQueue&
-    getTxAdvertQueue()
-    {
-        return mTxAdvertQueue;
-    };
+    bool peerKnowsHash(Hash const& hash);
+    void clearBelow(uint32_t ledgerSeq);
```

Comment:
Please document these public functions

Reply:
Done. Note on `peerKnowsHash`: I actually removed this method from the public API, so now the decision to broadcast an advert is isolated inside the Peer class.